databases/week1/week1.txt

```diff
@@ -0,0 +1,32 @@
+1)Find out how many tasks are in the task table
+--SELECT count(id) AS count_task  FROM task;
+
+2)Find out how many tasks in the task table do not have a valid due date
+--SELECT count(id) AS  "number of tasks with invalid due date" FROM task WHERE due_date IS null;
+
+3)Find all the tasks that are marked as done
+--SELECT * FROM task WHERE status_id = 3;
+
+4)Find all the tasks that are not marked as done
+--SELECT * FROM task WHERE status_id != 3;
+
+5)Get all the tasks, sorted with the most recently created first
+--SELECT * from task ORDER BY CREATED DESC ;
+
+6)Get the single most recently created task
+--SELECT * from task ORDER BY CREATED DESC LIMIT  1 ;
+
+7)Get the title and due date of all tasks where the title or description contains database
+--SELECT title,due_date FROM task WHERE title LIKE '%DATABASE%' or DESCRIPTION LIKE '%DATABASE%';
```

Comment:
These 3 queries work, but you should be careful to write the column names exactly like they are defined, ie. using all lowercase letters in this case.